spark-tensorflow/trainer/task.py

```diff
@@ -17,8 +17,8 @@
 from __future__ import print_function
 
 import argparse
-import data
-import model
+from . import data
```

Comment:
These import changes had broken the tensorflow tests (`test-tf.sh`).

Proposed a fix in [this PR](https://github.com/GoogleCloudPlatform/cloud-dataproc/pull/31).